CHANGELOG.md

```diff
@@ -31,9 +54,6 @@ dropping support for dbt versions before 1.0.0. This leverages functionality fro
 functionality supported directly by SQLFluff, and performance during testing of the new
 version has been reported as significantly faster.
 
-There will likely be more changes to rule classification before a full release of 2.0.0,
```

Comment:
Why was this removed?

Reply:
I just moved it up to the above section. It seemed like it's only a useful warning if this is the most recent release. When we do the full 2.0.0 release I'll write something about what those rule changes actually are.